dataverse-webapp/src/main/webapp/createEditDataverse.xhtml

```diff
@@ -420,29 +409,31 @@
                              widgetVar="allowMessagesBanners">
                         <div class="row">
                             <div class="col-sm-12 form-group">
-                                <label class="col-sm-3 control-label"><h:outputText
+                                <label class="col-md-3 control-label"><h:outputText
                                         value="#{bundle['dataverse.config.allow.messages.banners']}"/></label>
-                                <div class="col-sm-9">
-                                    <label class="metadata-blocks-default" for="dvAllowMessagesBanners">
-                                        <h:selectBooleanCheckbox id="dvAllowMessagesBanners" tabindex="7"
+                                <div class="col-md-9">
+                                    <label class="metadata-blocks-default" for="createEditDataverseForm:dvAllowMessagesBanners">
+                                        <h:selectBooleanCheckbox id="dvAllowMessagesBanners" 
```

Comment:
I can't alt+tab into this and every checkbox on this page.

Reply:
Fixed.
This bug was due to hiding default checkboxes with `visibility: hidden;` which as it turns out is not ARIA-compliant, as these nodes are ignored when tabbing. I've changed that to `opacity: 0;`.